src/libexpr/eval.hh

```diff
@@ -580,7 +580,7 @@ struct EvalSettings : Config
         "Whether builtin functions that allow executing native code should be enabled."};
 
     Setting<Strings> nixPath{
-        this, getDefaultNixPath(), "nix-path",
+        this, {}, "nix-path",
         "List of directories to be searched for `<...>` file references."};
```

Comment:
Also something not new from this PR, but the documentation of this options does't really reflect the default value (which depends on the value of `pure-eval` and `restrict-eval`)

Reply:
I've added a paragraph.

The documentation around the Nix path is kind of strewn about: there's [`nix-path`](https://nixos.org/manual/nix/unstable/command-ref/conf-file.html#conf-nix-path), [`NIX_PATH`](https://nixos.org/manual/nix/unstable/command-ref/env-common.html#env-NIX_PATH), the [`-I`](https://nixos.org/manual/nix/unstable/command-ref/opt-common.html#opt-I) option in "Common Options", and then each command that evaluates things (e.g. [`nix eval`](https://nixos.org/manual/nix/unstable/command-ref/new-cli/nix3-eval.html)) has its own copy of the actual documentation for `--include`/`-I` (which is the most extensive)...

cc @fricklerhandwerk

Reply:
Ah, and the actual documentation for [paths](https://nixos.org/manual/nix/unstable/language/values.html#type-path) only mentions `NIX_PATH`.

Reply:
Yeah, that's a bit scattered all around :grimacing:

But thanks any way